lsb_release

```diff
@@ -192,7 +194,7 @@ InitDistribInfo() {
 	if [ -n "$MSG_DISTRIBUTOR" ]; then
 	    DISTRIB_ID=$MSG_DISTRIBUTOR
 	else
-	    case "$PRETTY_NAME" in
+	    case "$NAME" in
```

Comment:
This breaks all SLE and ALP products from SUSE

Reply:
I just pushed a commit that should fix that.